doc/source/advanced/datatype.rst

```diff
@@ -0,0 +1,90 @@
+***********************************
+Data Types in Lux
+***********************************
+
+.. note:: You can follow along this tutorial in a Jupyter notebook. 
+
+Currently, Lux supports four data types:
+
+* Quantitative
+* Temporal
+* Nominal
+* ID
+
+Note that these data types are different from Pandas `dtype`. These data types are automatically inferred based on the `dataframe` inputted.
+The following code snippet shows what data types were inferred:
+
+.. code-block:: python
+
+  df = pd.read_csv("https://github.com/lux-org/lux-datasets/blob/master/data/college.csv?raw=true")
+  df.data_type
+
+
+Quantitative
+------------
+The Quantitative data type is used when there is a count or measure of a certain attribute. 
+In the example above, the column `AcceptanceRate` is quantitative because it is a measure. 
+Also, any aggregate such as means and medians will be categorized as quantitative. 
+Usually, Lux is able to compute Correlations between two quantitative columns or show the distribution of each column.
+
+.. image:: https://github.com/jinimukh/lux-resources/blob/datatype/doc_img/datatype-1.png?raw=true
+  :width: 700
+  :align: center
+  :alt: Displays correlation of quantitative variables.
+
+Nominal
+--------
+The Nominal data type is for categorical data. 
+For example, `PredominantDegree` is nominal because rather than being an explicit measure, it describes an attribute. 
+In this case, there are three possible values: `Associate`, `Bachelor's`, and `Certificate`.
+Lux displays these variables under the `Occurrence` tab as bar charts for the number of occurrences. 
+Upon more granular searches by specifying intent, stacked bar charts comparing two nominal variables is also possible.
+
+.. image:: https://github.com/jinimukh/lux-resources/blob/datatype/doc_img/datatype-2.png?raw=true
+  :width: 700
+  :align: center
+  :alt: Displays bar chart for nominal variables.
```

Comment:
Can we add info about `ordinal` data types too? We should add support for this also.

Reply:
Technically, we don't support ordinal data type, I believe. I'll add this when we add support?

Reply:
Yeah, let's defer this to later